lib/modules/dosomething/dosomething_api/resources/campaign_resource.inc

```diff
@@ -359,6 +362,23 @@ function _campaign_resource_reportback($nid, $values) {
     watchdog('dosomething_api', '_campaign_resource_reportback:' . json_encode($values));
   }
 
-  return dosomething_reportback_save($values, $user);
+  $transaction_id = $_SERVER['HTTP_X_REQUEST_ID'];
+
+  if (! $transaction_id) {
```

Comment:
yeah we did something similar, but perhaps more apparent with NS/Phoenix there was a query param `?create_drupal_user` on NS and [`forward`] (https://github.com/DoSomething/phoenix/commit/184063973871bc4d4296c73db19c2bbbb3f79215#diff-1492ad29ad6cef475c92f6ed9d58a7ebL34) on phoenix (which was just removed) to allow this.